cmds/ocm/pkg/processing/buffer.go

```diff
@@ -365,6 +369,9 @@ func (ob *orderedBuffer) SetFrame(frame BufferFrame) {
 }
 
 func (ob *orderedBuffer) Add(e ProcessingEntry) bool {
+	// This handler here prevents the calling function's lock to be not released.
```

Comment:
Sure. But that is super difficult in some of the cases because the functions are called either recursively or through many indirections. It's impossible to tell where the snake begins. :)

This particular function appears to be called from the parallel executor if I see it correctly. Which is called from a bunch of things that I was maybe able to track back to some cli setup. So where do you think would be a good place to catch this one? :)